packages/webapp/package.json

```diff
@@ -138,7 +138,7 @@
     "i18next-parser": "^3.6.0",
     "jest-environment-jsdom-fourteen": "1.0.1",
     "jest-resolve": "24.9.0",
-    "jest-watch-typeahead": "^0.6.5",
+    "jest-watch-typeahead": "^1.0.0",
```

Comment:
The tests won't run with that version.
![screenshot-6bf5d6db](https://user-images.githubusercontent.com/21227171/146131339-97462c5b-283c-4bbe-83cf-b24ba406a318.jpg)

Reply:
Couldn't find a fix. Rollback to 0.6.5